### DISCUSSION TITLE
Setting which model you use when using the openai api

### DISCUSSION THREAD

**Discussion by thoughtfission** [bookfere/Ebook-Translator-Calibre-Plugin#235] [Q&A]
Hi,

Is there a way to set the model you use when connecting to OpenAI? Currently when I connect, it uses the 3.5 model which is proving to be glacially slow. I would like to connect to 4.5 Turbo.

Any advice or guidance would be much appreciated.

Thanks

**Comment by bookfere** [comment]
In the "**Tune ChatGPT**" section of Setting, you have the option to add a custom model.

**Comment by thoughtfission** [comment]
Thanks!
